<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotifyPage</class>
 <widget class="QFrame" name="NotifyPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLable{
color:orange;
font-size:36px;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="mpBgFrame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>238</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#mpBgFrame {
	border-image: url(:/image/img_tipsbar_bg.png);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>23</number>
      </property>
      <property name="bottomMargin">
       <number>23</number>
      </property>
      <item row="0" column="2">
       <widget class="QLabel" name="mpNotifyLab">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>信息提示 </string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="mpNameLab">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="mpIdLab">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="QLabel" name="mpHeadImg">
        <property name="minimumSize">
         <size>
          <width>192</width>
          <height>192</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
